src/project.ts

```diff
@@ -82,15 +82,22 @@ export class ProjectModule {
         return this.xhr
             .get("/gdc/app/account/bootstrap")
             .then(r => r.getData())
-            .then((result: any) => {
-                const currentProject = result.bootstrapResource.current.project;
-                // handle situation in which current project is missing (e.g. new user)
-                if (!currentProject) {
-                    return null;
-                }
+            .then(this.getCurrentProjectIdInBootstrap);
+    }
 
-                return result.bootstrapResource.current.project.links.self.split("/").pop();
-            });
+    /**
+     * Return current project id in bootstrap
+     * @method getCurrentProjectIdInBootstrap
+     * @param bootstrapData - data was got from bootstrap resource
+     */
+    public getCurrentProjectIdInBootstrap(bootstrapData: any) {
+        const currentProject = bootstrapData.bootstrapResource.current.project;
+        // handle situation in which current project is missing (e.g. new user)
+        if (!currentProject) {
+            return null;
+        }
+
+        return bootstrapData.bootstrapResource.current.project.links.self.split("/").pop();
     }
```

Comment:
For consistency, please consider to 
- Move thin function into `src/bootstrap.ts` or 
- Move `getMapboxToken` out of `src/bootstrap.ts`

They have the same functionality. 
I prefer the second option :)